shinkai-bin/shinkai-node/tests/it/db_job_tests.rs

```diff
@@ -494,345 +427,6 @@ mod tests {
         assert_eq!(job_message_4.content, "Hello World 4".to_string());
     }
 
-    #[tokio::test]
```

Comment:
@benolt these test include step history but they are also testing the tree structure of the messages, we shouldnt remove the entire test